### ISSUE TITLE
Responsible Disclosure Policy Missing

### ISSUE THREAD

**Issue by nothingismagick** [shopware/platform#188]
I don't want to preach to the choir, but since I saw that with v6 you are moving toward a more open community, I thought I would leave this feedback here for the maintainers.

One of the things that is very important in both open source and in projects that use the node (because of your vue deps) and php stacks (in the backend) and docker (as a container) is having a policy about responsible disclosure for security vulnerabilities in each and every repo - or failing that an organization wide policy that is referenced and available.

Without a point of contact, you run the serious risk that less security savvy developers might accidentally disclose issues with your codebase. Here is a flow of responsible disclosure:

## What does Responsible Disclosure mean?
When a security researcher discovers a vulnerability, the generally accepted flow of responsible disclosure is as follows:

| WHO        | WHAT                                                        | HOW        |
|------------|-------------------------------------------------------------|------------|
| Researcher | **DISCOVER** a vulnerability                                | Privately  |
| Researcher | **CONSULT** the Security Disclosure Guidelines              | Privately  |
| Researcher | **NOTIFY** code-owners of a purported vulnerability         | Privately  |
| Code-Owner | **TRIAGE** the code to prove that the vulnerability exists  | Internally |
| Code-Owner | **PATCH** the vulnerability                                 | Internally |
| Researcher | **VALIDATE** that the patch actually solves the issue       | Privately  |
| Code-Owner | **RELEASE** a new version of the code with the patch        | Publicly |
| Researcher | **PUBLISH** the write-up of the vulnerability               | Publicly |
| Consumer   | **UPGRADE** to the latest version of the safer code         | Privately  |

Here is an example of a good disclosure (as recommended by @lirantal from SNYK)
https://github.com/verdaccio/verdaccio/blob/master/SECURITY.md that came up in this discussion: https://github.com/sfosc/sfosc/issues/85

**Comment by nothingismagick** [comment]
This issue is a perfect example of what I mean.
https://github.com/shopware/platform/issues/186

It references a potential exploit and should never land in the public issue queue before it is triaged and resolved.

**Comment by nothingismagick** [comment]
This would be a viable place to host your policy:
https://github.com/shopware/platform/security/policy

And you should consider managing a profile here:
http://hackerone.com

**Comment by jenskueper** [comment]
Hello nothingismagick,

thank you very much for raising this issue. I have updated our README.md and created a SECURITY.md file. These changes should be visible in a few minutes.

We are currently still in an early access phase, but we might introduce a bug/security bounty program in the future. I have created a ticket in our issue tracker regarding this: https://issues.shopware.com/issues/NEXT-4898

Best regards
Jens
